modules/util.py

```diff
@@ -1046,7 +1070,7 @@ def __repr__(self):
         return super().__repr__()
 
 
-def get_matching_config_files(config_pattern: str, default_dir: str) -> list:
+def get_matching_config_files(config_pattern: str, default_dir: str):
```

Comment:
```suggestion
def get_matching_config_files(config_pattern: str, default_dir: str) -> list:
```
Please keep the type annotations, I'm not sure what syntax highlighting errors you are referring to, can you explain further?